src/AtomBeautify.ts

```diff
@@ -130,9 +114,9 @@ export class AtomBeautify {
       // TODO
     }
 
-    private getConfigFromSettings(language: string) {
+    private configFromSettings(language?: Language) {
       const config = atom.config.get("atom-beautify");
-      return _.get(config, language.toLowerCase());
+      return language ? _.get(config, language.name) : config;
```

Comment:
I think `config` and `config[language.name]` are very different types.

For example:

```
const config = {
  "JavaScript": {
    "indent_size": 2
  }
};
```